engine.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any
+from .store import Store
+from abc import ABCMeta
+from abc import ABC
+from abc import abstractmethod
+
+
+class StorageEngine(ABC):
```

Comment:
Please add comments describe the class 
http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html